packages/typespec-test/test/anomalyDetector/generated/openapi/v1.1/openapi.json

```diff
@@ -658,7 +658,7 @@
       ],
       "x-ms-enum": {
         "name": "DataSchema",
-        "modelAsString": true,
+        "modelAsString": false,
```

Comment:
enum will always be treated as fixed enum https://azure.github.io/typespec-azure/docs/next/release-notes/release-2024-06-11#azure-toolstypespec-autorest-2